syntax="proto2";
package sps;

option cc_generic_services = true;

message HttpRequest {};
message HttpResponse {};

service PushService {
    rpc subscribe(HttpRequest) returns (HttpResponse);
    rpc subsist(HttpRequest) returns (HttpResponse);
};
